src/resolver/cresolver.cc

```diff
@@ -4,8 +4,18 @@
 
 #include "api/hustle_db.h"
 #include "catalog/catalog.h"
+#include "execution/execution_plan.h"
+#include "operators/select.h"
+#include "operators/utils/operator_result.h"
 #include "resolver/select_resolver.h"
 
+std::shared_ptr<hustle::storage::DBTable> execute(
```

Comment:
Do you need these forward declarations? 
From what I can see in this file you could reorder the function implementations and remove them.

Reply:
Sure!